content/en/docs/topics/styles/index.md

```diff
@@ -460,20 +460,17 @@ environment variable or the `dicpath` key.
 
 `spelling` offers two different ways of ignoring non-dictionary words:
 
-1. Using *ignore* files: Ignore files are plain-text files
-   that list words to be ignored during spell check (one case-insensitive entry
-   per line) . For example,
+- Using plain-text *ignore* files that list words to be ignored during spell check. You can name these files anything you'd like and add one case-insensitive entry
```

Comment:
Is this a numbered list (procedure) to show the importance? For me, it's bullet-point list.